README.md

```diff
@@ -3,70 +3,88 @@
 [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-908a85?logo=gitpod)](https://gitpod.io/#https://github.com/gitpod-io/workspace-images)
 [![Build from Main](https://github.com/gitpod-io/workspace-images/actions/workflows/push-main.yml/badge.svg)](https://github.com/gitpod-io/workspace-images/actions/workflows/push-main.yml)
 
-Ready-to-use Docker images for [gitpod.io](https://www.gitpod.io) workspaces. All images are available on Docker Hub under [gitpod/*](https://hub.docker.com/u/gitpod).
-For examples on how to use these images please take a look at the [documentation](https://www.gitpod.io/docs/config-docker#configure-a-public-docker-image).
+Ready-to-use Docker images for [gitpod.io](https://www.gitpod.io) workspaces. All images are available on [Gitpod's Dockerhub page](https://hub.docker.com/u/gitpod).
+
+For an example of how to use these images, please take a look at the [documentation](https://www.gitpod.io/docs/configure/workspaces/workspace-image#configure-a-public-docker-image).
 
 ## 📢 Announcements
 
 We upgraded to support [dazzle v2](https://github.com/gitpod-io/dazzle) and build with GitHub Actions.
 
-### 📷 Images we'll upgrade
+## Summary
+
+- **Operating System**: Ubuntu 20.04.4 LTS
+- **OS Type**: Linux
+- **Architecture**: x86_64
+
+## Images
+
+By default, Gitpod uses a standard Docker Image called [`workspace-full`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L23) as the foundation for workspaces.
+
+Workspaces started based on this default image come pre-installed with Docker, Nix, Go, Java, Node.js, C/C++, Python, Ruby, Rust & PHP as well as tools such as Homebrew, Tailscale, Nginx & several more.
+
+If this image does not include the tools you need for your repository, we recommend you start with [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3) and customize it according to your needs. You can refer to this [document to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
+
+OR, [Bring your Dockerfile](https://www.gitpod.io/docs/configure/workspaces/workspace-image#configure-a-public-docker-image).
```

Comment:
This points to the same document as the one in the sentence above. Can this be removed?